molsets.egg-info/SOURCES.txt

```diff
@@ -0,0 +1,47 @@
+MANIFEST.in
```

Comment:
Remove molsets.egg-info from PR—this is a temporary directory, you should not commit it to the repository